stack/eslint-plugin-packlets/package.json

```diff
@@ -19,7 +19,8 @@
     "build": "heft test --clean"
   },
   "dependencies": {
-    "@rushstack/tree-pattern": "workspace:*"
+    "@rushstack/tree-pattern": "workspace:*",
+    "@typescript-eslint/experimental-utils": "3.4.0"
```

Comment:
@sargunv What happens if the consuming project is using a different (e.g. newer) version of `@typescript-eslint/experimental-utils`?  In that case two versions of `@typescript-eslint/experimental-utils` will get installed side-by-side.

I wonder if we should loosen up the version range to `^3.4.0`?

Reply:
Added the caret range specifier

Reply:
Noticed there's a constraint that the version needs to be consistent across the repo, so I updated it to use `^` in the other packages too, and fixed the unlisted dependency in the other two eslint plugin packages. However, now I'm getting the following on `rush install`, so I assume CI will fail too:

```
ERROR: Internal Error: Cannot find shrinkwrap entry dependency "@typescript-eslint/experimental-utils" for workspace
project: @rushstack/eslint-plugin
You have encountered a software defect. Please consider reporting the issue to the maintainers of this application.
```